File: macroslib/tests/expectations/inline_function.rs
foreigner_class!(class BLAUtils {
    fn latitude_to_str(lat: Option<f64>, plus_sym: &str, minus_sym: &str)
                       -> String {
        format!(
            "{}",
            Latitude::new_with_symbols(lat, plus_sym.into(), minus_sym.into())
        )
    }
    fn longitude_to_str(lon: Option<f64>, plus_sym: &str, minus_sym: &str)
                        -> String {
        format!(
            "{}",
            Longitude::new_with_symbols(lon, plus_sym.into(), minus_sym.into())
        )
    }
});

File: macroslib/tests/expectations/inline_function.java
r#"public final class BLAUtils {

    public static native @NonNull String latitude_to_str(@Nullable Double lat, @NonNull String plus_sym, @NonNull String minus_sym);

    public static native @NonNull String longitude_to_str(@Nullable Double lon, @NonNull String plus_sym, @NonNull String minus_sym);

    private BLAUtils() {}
}"#;
